src/main/java/com/redhat/devtools/lsp4ij/settings/ui/AbstractServerMappingTablePanel.java

```diff
@@ -44,6 +44,7 @@ public AbstractServerMappingTablePanel(ColumnInfo<ServerMappingSettings, String>
                 new ArrayList<>(),
                 0);
         table = new TableView<>(model);
+        table.setColumnSelectionAllowed(true);

```

Comment:
Provides a better experience (IMO) when selecting a specific cell for editing after adding a new row.